test.js

```diff
@@ -446,6 +450,10 @@ test('GitHub.com URLs', urlMatcherMacro, new Map([
 		'https://togithub.com/fregante/shorten-repo-url/pull/32/files',
 		'#32 (files)',
 	],
+	[
+		'https://togithub.com/nodejs/node/pull/123/files',
+		'nodejs/node#123',
```

Comment:
This should have failed but it didn't… There's probably a bug lurking somewhere.

Reply:
Never mind, forgot this was a `Map` and thus values with the same keys are overwritten :sweat_smile: